src/NextjsBuild.ts

```diff
@@ -63,7 +63,7 @@ export class NextjsBuild extends Construct {
    * Cache directory for generated data.
    */
   public get nextCacheDir(): string {
-    const dir = path.join(this.getNextBuildDir(), NEXTJS_CACHE_DIR);
+    const dir = path.join(this.getNextBuildDir(), CACHE_BUCKET_KEY_PREFIX);
```

Comment:
Shouldn't this remain `NEXTJS_CACHE_DIR`? See folder structure below:
![image](https://github.com/jetbridge/cdk-nextjs/assets/35735118/cb77da45-fdda-4ddb-9703-63f9121b0def)

Reply:
Honestly I don't know, I was a bit torn... you're probably right though. I kept `NEXTJS_CACHE_DIR`. I'm also happy to keep both, this is one of those times when I say, "I wonder if I hadn't gotten bored while reading [Code Complete](https://www.oreilly.com/library/view/code-complete-2nd/0735619670/) if I'd just know 100% the right way to go"